apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -0,0 +1,195 @@
+import { QuestionCircleOutlined } from '@ant-design/icons'
+import { BuildingSection, NewsItemScope, Property as PropertyType, Resident as ResidentType } from '@app/condo/schema'
+import styled from '@emotion/styled'
+import compact from 'lodash/compact'
+import difference from 'lodash/difference'
+import every from 'lodash/every'
+import filter from 'lodash/filter'
+import intersection from 'lodash/intersection'
+import map from 'lodash/map'
+import uniq from 'lodash/uniq'
+import uniqBy from 'lodash/uniqBy'
+import React from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { useOrganization } from '@open-condo/next/organization'
+import { Card, Space, Typography } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+import { Tooltip } from '@condo/domains/common/components/Tooltip'
+import { queryFindResidentsByNewsItemAndScopes } from '@condo/domains/news/utils/accessSchema'
+import { Property } from '@condo/domains/property/utils/clientSchema'
+import { Resident } from '@condo/domains/resident/utils/clientSchema'
+
+interface CounterProps {
+    label: string
+    value: number
+    hint?: string
+}
+
+const Counter: React.FC<CounterProps> = ({ label, value, hint }) => (
+    <Space direction='vertical' align='center' size={8}>
+        <Space size={8} direction='horizontal' align='start'>
+            <Typography.Title level={3} type='success'>{value}</Typography.Title>
+            {hint && (
+                <Tooltip
+                    title={hint}
+                    placement='bottom'
+                    children={<QuestionCircleOutlined style={{ color: colors.gray['5'] }}/>}
+                />
+            )}
+        </Space>
+        <Typography.Text type='secondary'>{label}</Typography.Text>
+    </Space>
+)
+
+const JustifiedSpace = styled(Space)`
+  justify-content: space-evenly;
+`
+
+const isTargetedToOrganization = ({ property, unitType, unitName }: NewsItemScope) => (
+    !property && !unitType && !unitName
+)
```

Comment:
Does not make sense now. No scopes means the whole organization.

Reply:
Maybe to require value for at least one field (`property`, `unitType`, `unitName`) for `NewsItemScope`?

Reply:
Yes, such validation will be useful.

Reply:
Done